VTUL/dlp-access
# fix vulnerabilities

**JIRA Ticket**: (https://webapps.es.vt.edu/jira/browse/LIBTD-2118) (:star:)

# What does this Pull Request do? (:star:)
Fix npm vulnerabilities from 38935 to 2 

# How should this be tested?
1. Verify results in Amplify console
2. Test site functions 

Note: Temporary disable disableBodyScroll function which causes app break. Will create another ticket for it. 

# Interested parties
Tag (@tingtingjh ) interested parties

(:star:) Required fields
